### PR TITLE
Rename to Red Hat OpenShift Local

### PR DESCRIPTION
**Pull request by gbraad**

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>CodeReady Containers</title>
+    <title>OpenShift Local</title>
```
Comment (gbraad) [inline frontend/public/index.html]: Should this rather be `Red Hat OpenShift Local` ?

@cfergeau @robin-owen @anjannath WDYT ?

Comment (anjannath) [reply frontend/public/index.html]: for upstream so only `OpenShift Local` is better i think

File: frontend/src/windows/SplashWindow.tsx [hunk 2]
```diff
@@ -10,22 +10,22 @@ import {
...
-import logo from '../assets/crc-logo-white.png';
+import logo from '../assets/openshift-local.png';
 import { useNavigate } from 'react-router-dom';
 
 export default class SplashWindow extends React.Component {
   render() { 
     return (
       <Page>
-        <PageSection sticky="top" variant={PageSectionVariants.darker} isWidthLimited hasShadowBottom={false} style={{height:"420px"}}>
+        <PageSection sticky="top" variant={PageSectionVariants.light} isWidthLimited hasShadowBottom={false} style={{height:"420px"}}>
```
Comment (anjannath) [inline frontend/src/windows/SplashWindow.tsx]: `.darker` is no good or worse then `.light` ?

Comment (gbraad) [reply frontend/src/windows/SplashWindow.tsx]: Changed the image to our own version with transparency

**Comment by anjannath** [comment]
image

image

image

**Review by anjannath — APPROVED** [review]

**Pull request merged**